testsuite/MDAnalysisTests/topology/test_top.py

```diff
@@ -36,13 +36,10 @@
     PRMErr1,
     PRMErr2,
     PRMErr3,
-    PRM_UreyBradley
+    PRM_UreyBradley,
+    PRM19SBOPC
```

Comment:
```suggestion
    PRM19SBOPC,
```

Just helps the next time we need to add a new file